README.md

```diff
@@ -369,6 +395,35 @@ The `iotedgedev` module has the following commands:
 `iotedgedev project --help`
 - `--create TEXT`  Creates a new Azure IoT Edge project. Use `--create .` to create in current folder. Use `--create TEXT` to create in a subfolder.
 
+
+**azure**
```

Comment:
For this section, please copy and paste text from the actual -h command output,, so they are exactly the same.

Reply:
the `--resource-group-location` is a choice parameter with a long list of locations. LMK if you want that in

Reply:
No, it's fine as is.